data/scenarios/Testing/1379-single-world-portal-reorientation.yaml

```diff
@@ -0,0 +1,101 @@
+version: 1
```

Comment:
This file needs to be added to both `00-ORDER.txt` and also to the integration tests.